### ISSUE TITLE
Small focus inconsistency in dialogs with sections

### ISSUE THREAD

**Issue by alecpl** [tinymce/tinymce#5228]
In TinyMCE 5.1.1 when you open Emoticons dialog the search input is focused. However, when you open Special character dialog the focus is initially set on the first nav-item ("All") element.

Additional issue, in both dialogs when the search input is focused and you move mouse over special character/emoticon the focus will be grabbed from the input. Not a big issue, but imo hover operations should not grab focus from inputs.

**Comment by lnewson** [comment]
Yep, that's definitely a bug that looks like it's existed since 5.0 GA. TinyMCE 5 is meant to try and focus the first focusable element (the tab nav item in this case) but plugins, etc... can override that if needed, which is what the emoticons dialog is doing. I'll log a bug to get it fixed, thanks @alecpl!

**Comment by dhavalwd** [comment]
Hello @lnewson , thanks for looking into this issue. I am having similar issue with `styleselect` toolbar (or event `formatselect` toolbar option). I have this [codepen](https://codepen.io/cooldhavs/pen/qBjxoWY) here to demo the issue.

Basically, whenever user clicks on the `styleselect` dropdown or `formatselect` dropdown, and hovers over the different options, tinymce moves the focus to the next element which I am assuming is a bug. Let me know if you want me to create a separate bug for this but I thought that my issue is very much aligned with this one hence commenting it here.

Thanks in advance.

**Comment by lnewson** [comment]
Hi @dhavalwd,

That is a separate issue to this, however I can confirm that is expected behaviour and is required for accessibility purposes to allow  for someone to mouse over an item and then still switch to the keyboard to navigate further.

**Comment by lnewson** [comment]
As for the original issue, we forgot to make an update here sorry, however the first case reported (the inconsistent focusing) was fixed back in TinyMCE 5.1.5.

**Comment by dhavalwd** [comment]
Thank you @lnewson for the quick response. As this is a feature, please ignore my comment :)
